guide/ops/catalog/index.md

```diff
@@ -126,6 +126,33 @@ The following optional catalog metadata is supported:
   and if entities have been deployed against that version, their behavior may change in subtle or potentially incompatible ways.
   To avoid this situation, it is highly recommended to use OSGi version stamps as part of the URL.
 
+#### Locations in Catalog
```

Comment:
I would move this section to the end of the 'Catalog YAML Examples' section below, its just another example of a yaml catalog entry